Library/processmessage.py

```diff
@@ -8,33 +8,27 @@ def __init__(self):
 
     def processMessage(self, message):
         if message.find("[img]") >= 0:
-            message = message.split("]")[1].split("-")[0]
-            message = "Photo | https://ossendpoint.azureedge.net/download/fromkey/{0}".format(
+            message = message.split("]")[1].split("|")[0]
+            message = "Photo | https://probe.aiursoft.com/Download/InSites/{0}".format(
                 message)
             return message
 
         if message.find("[video]") >= 0:
             message = message.split("]")[1]
-            message = "Video | https://ossendpoint.azureedge.net/download/fromkey/{0}".format(
+            message = "Video | https://probe.aiursoft.com/Download/InSites/{0}".format(
```

Comment:
Hey! Don't forget to URL encode this! Otherwise when you upload a video called `girl@me.mp4` will cause it not able to download.